pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -0,0 +1,522 @@
+
+You can use bind mounts to transfer durability files such as snapshot or WAL
+files inside the container to restore data, or CSV files you will use to import
+data with `CSV LOAD` clause. 
+
+Bind mounts do not handle data permissions which could cause issues with
+permissions. 
+
+### Stop container
+
+<Callout type="info">
+Stopping the container isn't the same thing as [deleting it](#containers-and-data-persistence).
+</Callout>
+
+Database instances are stopped by stopping the Docker container with the command
+`docker stop`. To stop a container you need [to know the container's
+ID](#how-to-retrieve-a-docker-container-id) or [name](#container-name).
+
+You can list all the containers you want to stop in one `docker stop` command:
+
+```
+docker stop CONTAINER1_ID CONTAINER2_ID CONTAINER3_NAME
+```
+
+### Start container
+
+If you want to start a stopped container, list them using the following command: 
+
+```
+docker ps -a
+```
+
+Then start the container with: 
+```
+docker start <CONTAINER_ID> 
+```
+
+or 
+
+```
+docker start <CONTAINER_NAME> 
+```
+
+### Retrieve a Docker container ID
+
+Run the following command:
+
+```
+docker ps
+```
+
+You should get an output similar to this:
+
+```console
+CONTAINER ID   IMAGE                        COMMAND                  CREATED        STATUS        PORTS                                                                    NAMES
+45fa0f86f826   memgraph/memgraph-platform   "/bin/sh -c '/usr/bi…"   21 hours ago   Up 21 hours   0.0.0.0:3000->3000/tcp, 0.0.0.0:7444->7444/tcp, 0.0.0.0:7687->7687/tcp   admiring_almeida
+```
+
+You can shorten this ID to 4 letters if the ID remains unique, for example,
+`45fa`.
+
+### Retrieve a Docker container IP address
+
+To retrieve the Docker container IP address, first, you need [to retrieve its
+ID](#how-to-retrieve-a-docker-container-id) or use its [name](#container-name).
+
+Then run the following command if the container ID is `9397623cd87e`.
+
+```console
+docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' 9397623cd87e
+```
+
+### Browse files inside a Docker container
+
+To browse files inside a Docker container, first, you need [to retrieve its
+ID](#how-to-retrieve-a-docker-container-id) or use its [name](#container-name).
+
+Then run the following command if the container ID is `9397623cd87e`:
+
+```
+docker exec -it 9397623cd87e bash
+```
+
+To navigate through the container, use the following commands:
+
+- `ls` - list all the directories and files
+- `cd <directory>` - enter a directory
+- `cd ..` - leave the directory
+- `cat <file>` - list the content of a file
+
+You don't have to write file and directory names in full, once you write enough
+letters to have a unique string, press the `<TAB>` key.
+
+### Copy files from and to a Docker container
+
+To copy files from and to the Docker container, first, you need [to retrieve its
+ID](#how-to-retrieve-a-docker-container-id) or use its [name](#container-name).
+
+<Steps>
```

Comment:
I'm so sorry... I made a wrong suggestions here...

These can't be really steps because to copying the file from a container or to the container are not really steps 2 and 3, they are both steps 2 depending on what you need... But I also didn't like those number we had before

Do you have an idea how we could improve this? 

Maybe:

First, place yourself in the local directory where you want to copy the file. 

To **copy the file from the container to your current directory** use the following command:

```
docker cp <CONTAINER ID>:<path_in_container> <local_file_name>
```

Be sure to replace the `<CONTAINER ID>` parameter.

The example below will copy the configuration file to the user's Desktop:

```
C:\Users\Vlasta\Desktop>docker cp bb3de2634afe:/etc/memgraph/memgraph.conf memgraph.conf
```

To **copy the file from your current directory to the container** use the following command:

....... the rest of the content